datasets/Cargo.toml

```diff
@@ -14,6 +14,9 @@ ndarray-csv = "=0.5.1"
 csv = "1.1"
 flate2 = "1.0"
 ndarray-rand = { version = "0.14", optional = true }
+statrs = {version = "0.16.0", optional = true }
+rand = { version = "0.8", optional = true }
```

Comment:
`rand` is re-exported from `ndarray-rand`, so this dependency isn't needed

Reply:
gotcha removed.